scripts/entrypoint.sh

```diff
@@ -83,13 +95,21 @@ set_properties () {
     monetdbd set "logfile=$logfile" "$farm_dir"
     monetdbd set "snapshotdir=$snapshotdir" "$farm_dir"
     monetdbd set "snapshotcompression=$snapshotcompression" "$farm_dir"
+    for prop in "${farm_props[@]}"; do
+        if [[ ! "$prop" =~ ^listenaddr.*|^control.*|^passphrase.* ]]; then
```

Comment:
I think the pattern should include the = sign, that is, `listenaddr-.*`. This makes sure people cannot sneak in other (future) options that start with listenaddr, control or passphrase.
Also, I wonder if it would be more readable to use a case-statement:
```
case "$prop" in
    listenaddr=)
        ;;
    control=)
        ;;
    passphrase=)
        ;;
    *)
        continue
       ;;
esac
```
(untested)